static/app/routes.tsx

```diff
@@ -279,6 +279,27 @@ function buildRoutes() {
         )}
         key="org-join-request"
       />
+      {usingCustomerDomain && (
+        <Route
+          path="/relocation/"
+          component={errorHandler(withDomainRequired(OrganizationContextContainer))}
+          key="orgless-relocation"
+        >
+          <IndexRedirect to="welcome/" />
```

Comment:
This will still redirect to the getStarted page, going to keep this for now given we need another welcome page for later

Reply:
👍